odoo/addons/test_impex/tests/test_load.py

```diff
@@ -1255,3 +1255,167 @@ def test_unique_pair(self):
             m.group(2).split(', '),
             ['Value2', 'Value3']
         )
+
+class test_inherits(ImporterCase):
+    """ The import process should only assign a new xid (derived from the
+    childs') if the child is being created and triggers the creation of the
+    parent
+    """
+    model_name = 'export.inherits.child'
+    def get_xid(self, record):
+        return self.env['ir.model.data'].search([
+            ('model', '=', record._name),
+            ('res_id', '=', record.id),
+        ])
```

Comment:
As `get_xid` is always used with `mapped('complete_name')`, why not combining both?
```suggestion
    def get_xids(self, record):
        return self.env['ir.model.data'].search([
            ('model', '=', record._name),
            ('res_id', '=', record.id),
        ]).mapped('complete_name')
```

Reply:
Seemed unnecessary to fold it in there.

Reply:
Actually there is even an API for doing this
```python
get_xids(self, record):
    return record._get_external_ids()[record.id]
```